doc/facebook_insights.md

```diff
@@ -0,0 +1,50 @@
+# FacebookInsights connector
+
+Import data from facebook insights API.
+
+## Data provider configuration
+
+* `type`: `"MySQL"`
+* `name`: str, required
+* `host`: str, required
+* `user`: str, required
+* `db`: str, required
+* `password`: str
+* `port`: int
+* `charset`: str, default to utf8mb4
+* `connect_timeout`: int
+
+```javascript
+DATA_PROVIDERS: [
+  type:    'faceobok_insights',
+  name:    '<name>'
+]
+```
+
+## Data source configuration
+
+
+Either `query` or `table` are required, both at the same time are not supported.
```

Comment:
I think this comment comes from a copy paste, and is not related to Facebook Insights ;)